pantheon-bundle/src/main/resources/SLING-INF/content/docs/attributes.adoc

```diff
@@ -11,5 +11,6 @@
 
 // Publishing
 :PublishingPortal: Customer Portal
+:PublishingPortalSearh: Customer Portal Search
```

Comment:
Did you mean to include a `c` PublishingPortalSear**c**h?

Reply:
yes, typo!